test/types.d.ts

```diff
@@ -31,10 +31,13 @@ type ItFunction<STATE> = ((name: string, inner: (state: STATE) => Promise<void>)
     repeat(n: number): ItFunction<STATE>;
 };
 
-interface WorkerState {
+interface FixtureParameters {
```

Comment:
Somebody needs to put the matrix options in here.

Reply:
Not really, fixtures do that.